resources/wikia/modules/spec/imageServing.spec.js

```diff
@@ -21,8 +21,8 @@ describe('Image Serving', function () {
 		expect(typeof cutOptions.yOffset2).toBe('number');
 	});
 
-	using('image sizes', [
```

Comment:
Thank you @kenkouot I'll adapt this patten to jasmine API and add in to our docs on internal for reference.